src/psyclone/tests/psyGen_test.py

```diff
@@ -1051,9 +1050,8 @@ def test_call_multi_reduction_error(monkeypatch, dist_mem):
     with pytest.raises(GenerationError) as err:
         _ = PSyFactory("dynamo0.3",
                        distributed_memory=dist_mem).create(invoke_info)
-    assert(
-        "PSyclone currently only supports a single reduction in a kernel "
-        "or builtin" in str(err.value))
+    assert ("PSyclone currently only supports a single reduction in a kernel "
+            "or builtin" in str(err.value))
 
 
 def test_reduction_no_set_precision(dist_mem):
```

Comment:
Please could you also fix:

    psyGen_test.py:2232:35: R1732: Consider using 'with' for resource-allocating operations (consider-using-with)
    psyGen_test.py:2232:35: W1514: Using open without explicitly specifying an encoding (unspecified-encoding)

Reply:
Done